clang/lib/CodeGen/CGDebugInfo.h

```diff
@@ -355,12 +355,12 @@ class CGDebugInfo {
       llvm::ArrayRef<llvm::Metadata *> PreviousFieldsDI, const RecordDecl *RD);
 
   /// A cache that maps names of artificial inlined functions to subprograms.
-  llvm::StringMap<llvm::DISubprogram *> InlinedTrapFuncMap;
+  llvm::StringMap<llvm::DISubprogram *> InlinedSubprogramMap;
```

Comment:
Can you rename in a separate NFC patch?

Reply:
https://github.com/llvm/llvm-project/pull/132993